src/App.js

```diff
@@ -1,29 +1,14 @@
-import logo from './logo.svg';
```

Comment:
| Files to be deleted 🔴 |
|-|
| ![image](https://github.com/YoutLone/math-magicians/assets/93039359/cdd92ad2-171c-4990-8bd5-c1ebe086fdb0) |

- [ ] It would be nice if you remove all the boilerplate that got created by the CRA (texts, images, and styles) to make your project clean and follow the project requirements 👍